src/ostree/ot-builtin-gpg-sign.c

```diff
@@ -171,7 +171,7 @@ delete_signatures (OstreeRepo *repo,
 
       while (!g_queue_is_empty (&signatures))
         {
-          GVariant *child = g_queue_pop_head (&signatures);
+          child = g_queue_pop_head (&signatures);
           g_variant_builder_add_value (&signature_builder, child);
           g_variant_unref (child);
```

Comment:
Let's also rename this one rather than increasing the scope.  We could also use `g_autoptr()` here i.e.:
```suggestion
          g_autoptr(GVariant) sigchild = g_queue_pop_head (&signatures);
          g_variant_builder_add_value (&signature_builder, sigchild);
```

Reply:
Sure